test/primitives.jl

```diff
@@ -6,6 +6,9 @@
     @test coordtype(Point(1, 1)) == Float64
     @test coordtype(Point(1.0, 1.0)) == Float64
     @test coordtype(Point(1.0f0, 1.0f0)) == Float32
+    @test coordtype(Point(1u"m",1u"m")) == UNITFUL_FLOAT64
+    @test coordtype(Point(1.0u"m",1.0u"m")) == UNITFUL_FLOAT64
+    @test coordtype(Point(1.0f0u"m",1.0f0u"m")) == UNITFUL_FLOAT32
```

Comment:
Instead of checking the coordtype, what about just checking the `unit(coordinates(point))` and comparing with the expected value? The Unitful types are quite verbose and likely include implementation details that may change with time.

Reply:
Suggestion:
```julia
@test unit(coordtype(point)) == u"m"
@test Unitful.numtype(coordtype(point)) === Float64
```

Reply:
Thank you for the suggestions, @eliascrv and @juliohm. I'm looking for a function like `Unitful.numtype`. I will soon update the test code.